tests/PHPSA/ApplicationTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Tests\PHPSA;
+
+use PHPSA\Application;
+
+class ApplicationTest extends TestCase
+{
+    /**
+     * @covers \PHPSA\Application::__construct
+     */
+    public function testConstructor()
+    {
+        $application = new Application();
+
+        $this->assertInstanceOf('\PHPSA\Application', $application);
+        $this->assertInstanceOf('\Symfony\Component\Console\Application', $application);
```

Comment:
I dont known, for me it's not need
It's rly simple tests that cannot give nothing, about BC I dont thing that someone will extends from \PHPSA\Application, but maybe...